pt_PT.ini

```diff
@@ -10,52 +10,52 @@ Sound speed hack (DOA etc.) = Sound speed hack (Dead Or Alive, etc.)
 WASAPI (fast) = WASAPI (fast)
 
 [Controls]
-Analog Limiter = Analog limiter
-Analog Mapper High End = Analog mapper high-end (axis sensitivity)
-Analog Mapper Low End = Analog mapper low-end (inverse deadzone)
-Analog Mapper Mode = Analog mapper mode
-Analog Stick = Analog stick
+Analog Limiter = Limitador do analógico
+Analog Mapper High End = Mapeamento do analógico de alta qualidade (sensibilidade dos eixos)
+Analog Mapper Low End = Mapeamento do analógico de baixa qualidade (zona não coberta inversa)
+Analog Mapper Mode = Modo de mapamento do analógico
+Analog Stick = Analógico
 Auto = Auto
```

Comment:
I'm not exactly sure of the context either, but at worst `Auto` will only seem a little bit off imo.